src/styles/theme.scss

```diff
@@ -337,22 +411,40 @@ h5 {
   background-color: $primary-colour;
 }
 
+.card--alternate.card--with-icon {
+  background-color: transparent;
+  color: $secondary-colour;
+  border: 2px solid $icon-background;
+  border-radius: 10px;
+  article h2 {
+    color: $text-colour;
+  }
+
+}
+
 /* ------------------------------
 
     Info to help you
 
 ------------------------------ */
 
+.info {
+  background-color: transparent;
+  border: 2px solid $icon-background;
+  border-radius: 10px;
+  color: $text-colour;
+}
+
 .info:active,
 .info:focus,
 .info:hover {
-  background-color: $purple;
+  background-color: $yellow-light;
```

Comment:
I prefer these not to sneak in here - since if next person uses this brand as a template - we'll have a higher chance of non-brand colours sneaking in.

Reply:
you refer to the colour? Holly suggested lighter yellow for the hover colour on the info to help you cards. the pink - she reckoned - would not be such a good match for the hover state. This follows the look on hospice.
![Screenshot from 2020-01-10 17-35-47](https://user-images.githubusercontent.com/29477363/72173559-c257c400-33cf-11ea-9292-5a7ec4738e5e.png)

Reply:
The colour is ok. I's the variable I am referring to. They get assigned to a purpose in the var section and then re-used. I know you are following the previous implementation - and it's fine either way. The project is so small.